libcontainer/cgroups/utils_test.go

```diff
@@ -381,10 +396,8 @@ func TestIgnoreCgroup2Mount(t *testing.T) {
 }
 
 func TestFindCgroupMountpointAndRoot(t *testing.T) {
-	fakeMountInfo := `
-35 27 0:29 / /foo rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,devices
-35 27 0:29 / /sys/fs/cgroup/devices rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,devices
-`
+	fakeMountInfo := `35 27 0:29 / /foo rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,devices
+35 27 0:29 / /sys/fs/cgroup/devices rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,devices`
```

Comment:
this is fixed to not contain any extra empty lines (which is an issue with the mountinfo parser).

Reply:
@kolyshkin this still a problem? Can we fix that in the parser? (trim leading/trailing whitespace?)

Reply:
This was a problem with the test data, which is fixed.

I do not want to add extra trims/checks to the parser as those are entirely useless for real kernel-provided data, and I am trying to keep the parser fast and clean.

Reply:
Thx! I wondered if leading/trailing whitespace could occur in actual situations as well, but it that's not a possibility, then I agree with not stripping